torchtext/datasets/enwik9.py

```diff
@@ -22,10 +17,15 @@
 DATASET_NAME = "EnWik9"
 
 
-@_add_docstring_header(num_lines=NUM_LINES)
```

Comment:
Any reason we're removing the `_add_docstring_header` decorator and manually adding the docstrings? Can we modify the `_add_docstring_header` to work with a dataset function that doesn't have a `split` arg?

Reply:
The decorator is doing a job of avoiding writing doc-string and make strong assumptions about datasets. But it's scope is fairly limited (for eg: it does not handle cases when there are no splits or when we have additional arguments other than root and split). Overall, I think we probably need to get rid of _add_doctring_header decorator in light of this issue #1588 anyway since we plan to expand the scope of documentation of what we currently have.